src/components/Gameover/Gameover.js

```diff
@@ -9,12 +9,12 @@ const Gameover = ({ handleClick, isVisible }) => {
       }
     >
       <p className={styles.header}>
-        Gratulacje,
+        Congratulations,
```

Comment:
Can we change the` <br>` inside `<p>` tag to something like flex box block? Would be nicer probably.

Reply:
Sure, I'm going to fix that.